cosmos/lib/cosmos/conversions/segmented_polynomial_conversion.rb

```diff
@@ -146,7 +164,7 @@ def as_json
       @segments.each do |segment|
         params << [segment.lower_bound, segment.coeffs]
       end
-      { 'class' => self.class.name.to_s, 'params' => params }
+      { 'class' => self.class.name.to_s, 'params' => [params] }
```

Comment:
I would like to genericize segmented conversions as well, but that can be left for another day.